src/Wsdl/Event/RequestPostEvent.php

```diff
@@ -0,0 +1,101 @@
+<?php
+declare(strict_types = 1);
+
+namespace Skaut\Skautis\Wsdl\Event;
+
+use Serializable;
+use stdClass;
+
+class RequestPostEvent implements Serializable
+{
+
+    /**
+     * @var string Nazev funkce volane pomoci SOAP requestu
+     */
+    private $fname;
+
+    /**
+     * Parametry SOAP requestu na server
+     *
+     * @var array<int, mixed>
+     */
+    private $args;
+
+
+    /**
+     * @var float Doba trvani pozadvku
+     */
+    private $time;
+
+	  /**
+     * @var array|stdClass[]
+     */
+    private $result;
+
+    /**
+     * @param string $fname Nazev volane funkce
+     * @param array<int, mixed> $args  Argumenty pozadavku
+     * @param array|StdClass[] $result
+     */
+    public function __construct(
+      string $fname,
+      array $args,
+      $result,
+      float $duration
+    ) {
+        $this->fname = $fname;
+        $this->args = $args;
+        $this->result = $result;
+        $this->time = $duration;
+    }
+
+    public function serialize(): string
+    {
+        $data = [
+            'fname' => $this->fname,
+            'args' => $this->args,
+            'time' => $this->time,
+            'result' => $this->result,
+        ];
+        return serialize($data);
+    }
+
+    /**
+     * @param string $data
+     */
+    public function unserialize($data): void
+    {
+        $data = unserialize($data, [self::class, stdClass::class]);
```

Comment:
Viz https://www.php.net/manual/en/function.unserialize.php#refsect1-function.unserialize-parameters
```suggestion
        $data = unserialize($data, ['allowed_classes' => [self::class]]);
```